wowchemy/layouts/partials/page_metadata_authors.html

```diff
@@ -3,21 +3,54 @@
 {{- $taxonomy := "authors" }}
 {{ with .Param $taxonomy }}
   {{ $link_authors := site.Params.link_authors | default true }}
+
   {{ range $index, $name_raw := . }}
-    {{- $profile_page := site.GetPage (printf "/%s/%s" $taxonomy .) -}}
-    {{- $name := $profile_page.Title | default $name_raw -}}
     {{- if gt $index 0 }}, {{ end -}}
-    <span {{ if site.Params.highlight_superuser | and (eq $profile_page.Params.superuser true) }}class="font-weight-bold"{{end}}>
+
+    {{/* NOTE: Assumes that the "author name" matches their on-disk name. */}}
+    {{/* "slugify" the raw name to make lookups a bit easier. This likely needs "_" support as well.*/}}
+    {{- $lookup := replace $name_raw " " "-" -}}
+    {{- $target := printf "%s/%s" $taxonomy $lookup -}}
+
+    {{/* Check that profile page exists. If not, dont bother linking it. */}}
+    {{ $profile_page := cond (fileExists $target) (site.GetPage (printf "/%s" $target)) false }}
+
+    {{ $hl_members := false }}
+    {{ $hl_superuser := false }}
+    {{ $name := $name_raw }}
+
+    {{/* If $profile_page does not exist (e.g. $lookup is not on disk), search through author aliases. */}}
+    {{- if not $profile_page -}}
+      {{/* Try searching over known authors for their aliases. */}}
+      {{- range $author, $pages := site.Taxonomies.authors -}}
+        {{ $a := site.GetPage (printf "/%s/%s" $taxonomy $author )}}
+        {{- if and (not $profile_page) (isset $a.Params "aliases") -}}
+          {{- if in $a.Params.aliases $name_raw -}}
+            {{- $profile_page = $a -}}
+          {{- end -}}
+        {{- end -}}
+      {{- end -}}
+    {{- end -}}
+
+    {{ if $profile_page }}
+      {{ $name = $profile_page.Title }}
+      {{- $hl_members = site.Params.highlight_members -}}
+      {{- $hl_superuser = site.Params.highlight_superuser | and (eq $profile_page.Params.superuser true) -}}
+    {{- end -}}
+
+    <span {{ if or $hl_members $hl_superuser }}class="font-weight-bold"{{ end }}>
       {{- if and $profile_page $link_authors -}}
-        <a href="{{$profile_page.RelPermalink}}">{{$name}}</a>
+        <a href="{{$profile_page.RelPermalink}}">{{ $name }}</a>
       {{- else -}}
-        {{$name}}
+        {{ $name | title }}
```

Comment:
`| title`? Perhaps you meant to **default** to the **page param named Title**? You might want to check this code is performing what you intended.

Reply:
I'm intending for it to be "titlized". Mostly as a case for ensuring that the name is capitalized if not found on disk. (If users use `academic-admin` to export their bibtex, this is a wasteful operation; but I figured not everyone uses the tool and this guarantees capitalization on a name, which I presume folks would expect.